react/EmptyStateMessage.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC, Fragment } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { useCssHandles } from 'vtex.css-handles'
+
+import EmptyStateIcon from './components/EmptyStateIcon'
+
+const CSS_HANDLES = ['emptyStateIconWrapper', 'emptyStateMessage'] as const
+
+const EmptyStateMessage: FC = () => {
+  const handles = useCssHandles(CSS_HANDLES)
+
+  return (
+    <Fragment>
+      <div className={`${handles.emptyStateIconWrapper} tc pv8`}>
+        <EmptyStateIcon />
+      </div>
```

Comment:
Do we need a special block for this? Can we achieve the same result placing flex-layout + icon + rich-text?

Reply:
🤔 I'll try to achieve the same layout using these blocks. Should work!